.github/actions/commenter/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Copyright Strimzi authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+FROM node:slim
```

Comment:
Any reason why there is a dockerfile per action? Or could there be a simlink? Just thinking of saving upkeep

Reply:
Or have the actions refer to one dockerfile build?

Reply:
I haven't played around with it but actions are supposed to be their own little packages. I don't think we want to but these could each live in their own repo and we'd just change the location of where the action is pulled from